msal-dotnet-articles/acquiring-tokens/desktop-mobile/linux-broker-net.md

```diff
@@ -0,0 +1,89 @@
+
+Authentication brokers are **not** pre-installed on Linux but is bundled as a dependency of applications developed by Microsoft, such as [Company Portal](/mem/intune-service/user-help/enroll-device-linux). These applications are usually installed when a Linux computer is enrolled in a company's device fleet via an endpoint management solution like [Microsoft Intune](/mem/intune/fundamentals/what-is-intune). To learn more about Linux device set up with the Microsoft Identity Platform, refer to [Microsoft Enterprise SSO plug-in for Apple devices](/entra/identity-platform/apple-sso-plugin).
+
+## Dependency
+
+To use the broker, you will need to install a list of dependencies on Linux platform
+
+```bash
+libc++-dev
+libc++abi-dev
+libsecret-tools
+libwebkit2gtk-4.0
+```
+
+## Create a console app on Linux platform
+To use broker on Linux platform, you need to specify the `BrokerOptions` to `OperationsSystems.Linux`. Notice that we use the same option for both Windows Subsystem for Linux (WSL) and standalone Linux.
+```dotnet
+from msal import PublicClientApplication
+
+class Program
+    {
+        public static string ClientID = "your client id"; //msidentity-samples-testing tenant
+        public static string[] Scopes = { "User.Read" };
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
+
+            var pcaBuilder = PublicClientApplicationBuilder.Create(ClientID)
+                .WithAuthority("https://login.microsoftonline.com/common")
+                .WithDefaultRedirectUri()
+                .WithBroker(new BrokerOptions(BrokerOptions.OperatingSystems.Linux){
+                            ListOperatingSystemAccounts = true,
+                            MsaPassthrough = true,
+                            Title = "MSAL WSL Test App"
+                          })
+                .Build();
+
+            AcquireTokenInteractiveParameterBuilder atparamBuilder = pcaBuilder.AcquireTokenInteractive(Scopes);
+
+            AuthenticationResult authenticationResult = atparamBuilder.ExecuteAsync().GetAwaiter().GetResult();
+            System.Console.WriteLine(authenticationResult.AccessToken);
+        }
+    }
+```
+
+## Sample App 
+There is a sample app in MSAL.NET repo, it's under ./tests/devapps/WAM/NetWSLWam path. The sample app has a dependenc of `libx11-dev` package. Please run `apt install libx11-dev` to install the package. To run the sample app, just run the command below.
+```
+dotnet run --project tests\devapps\WAM\NetWSLWam\test.csproj
+```
+
+## WSL Scenario
+
+### Update to the latest version of WSL
+Please make sure you have updated to the most recent version of WSL. WAM Account Control dialog is supported from WSL version 2.4.13. Otherwise, the broker scenario won't work. Here is the command to update WSL.
```

Comment:
Specify a version instead of suggesting using the latest

```suggestion
Please ensure you are using WSL version 2.4.13 or higher, as the WAM Account Control dialog is supported starting from this version. Otherwise, the broker scenario won't work.
```